src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift

```diff
@@ -108,6 +122,18 @@ class ENAButton: DynamicTypeButton {
 	private func applyHighlight() {
 		highlightView.isHidden = !isHighlighted
 	}
+
+	func startSpinner() {
+		isEnabled = false
+		spinner.startAnimating()
+		applyStyle()
```

Comment:
`applyStyle` is redundant at this point, since it is already being called with `isEnabled`.